examples/DistributedGeneratorTest/DGTest.cpp

```diff
@@ -1,85 +1,84 @@
 
```

Comment:
Since we are checking for lines at the end of files and extra lines between code statements, do we want to suppress extra lines at the beginning of files as well? Do we care?

Reply:
We don't want extra lines lines at the beginning of files. We should probably put doxygen file documentation at the top of source files.